docs/modules/cloud/controller.md

```diff
@@ -0,0 +1,54 @@
+# Edge Controller
+
+
+## Edge Controller Overview
+ Controller(also known as edgecontroller) is the bridge between Kubernetes Api-Server and Edgecore
+
+   
+## Operations Performed By Edge Controller
+ 
+ The following are the functions performed by Edge controller :-
+  - Downstream Controller:Sync add/update/delete event to edgeCore from K8s Api-server
+  - Upstream Controller:sync watch and Update status of resource and events(node, pod and configmap) to K8s Api-server and also subscribe message from edgecore
+  - Controller Manager:Creates manager Interface which implements events for managing ConfigmapManager LocationCache and podManager
+
+
+### Downstream Controller:Sync add/update/delete event to edge
```

Comment:
When the document gets parsed for readthedocs all the headings with ## get added as a menu item in the table of contents and the sidebar menu.  
Use ## for any heading that  should be a part of the menu. Avoid long titles. 
If there is a longer title, keep a short name after ## and then a longer sub title under ###